pyke/lightcurve.py

```diff
@@ -79,12 +83,14 @@ def flatten(self, window_length=101, polyorder=3, **kwargs):
         trend_lc : LightCurve object
             Trend in the lightcurve data
         """
-        trend_signal = signal.savgol_filter(x=self.flux, window_length=window_length,
+        lc_clean = self.remove_nans()  # The SG filter does not allow NaNs
```

Comment:
I added a line to remove NaNs inside `flatten()` because `savgol_filter` seems to break when it is passed NaNs.  Not sure if this is the best approach.

Reply:
👍